{0} setUpClass (tempest.api.compute.volumes.test_attach_volume.AttachVolumeMultiAttachTest) ... SKIPPED: Neutron is not available
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_invalid_size [0.094482s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_zero [0.011694s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_without_passing_size [0.012350s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_delete_invalid_volume_id [0.082100s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_delete_volume_without_passing_volume_id [0.009495s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_get_volume_without_passing_volume_id [0.009352s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_volume_delete_nonexistent_volume_id [0.037636s] ... ok
{1} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_volume_get_nonexistent_volume_id [0.075941s] ... ok
{0} tempest.api.compute.volumes.test_volumes_get.VolumesGetTestJSON.test_volume_create_get_delete [2.840303s] ... ok
{1} tempest.api.volume.admin.test_backends_capabilities.BackendsCapabilitiesAdminTestsJSON.test_compare_volume_stats_values [1.066633s] ... ok
{1} tempest.api.volume.admin.test_backends_capabilities.BackendsCapabilitiesAdminTestsJSON.test_get_capabilities_backend [0.537765s] ... ok
{2} tempest.api.compute.volumes.test_volume_snapshots.VolumesSnapshotsTestJSON.test_volume_snapshot_create_get_list_delete [7.767313s] ... ok
{2} tempest.api.volume.admin.test_group_type_specs.GroupTypeSpecsTest.test_group_type_specs_create_show_update_list_delete [0.494309s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list [0.068754s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_param_limit [0.071468s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_param_offset_and_limit [0.157495s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_detail_param_limit [0.106258s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_detail_param_offset_and_limit [0.142294s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_details [0.069003s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_encrypted_volumes_extend.EncryptedVolumesExtendAttachedTest) ... SKIPPED: EncryptedVolumesExtendAttachedTest skipped as Neutron is not available
{1} tempest.api.volume.admin.test_groups.GroupsV314Test.test_create_group_from_group [11.875998s] ... ok
{2} tempest.api.volume.admin.test_groups.GroupsTest.test_group_create_show_list_delete [8.605355s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_reset_snapshot_status [0.282109s] ... ok
{2} tempest.api.volume.admin.test_groups.GroupsTest.test_group_update [12.099799s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_create_group_from_group_snapshot [15.261477s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_creating [3.589711s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_deleting [3.593214s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_error [3.654432s] ... ok
{2} tempest.api.volume.admin.test_groups.GroupsV320Test.test_reset_group_status [3.327257s] ... ok
{2} setUpClass (tempest.api.volume.admin.test_snapshot_manage.SnapshotManageAdminTest) ... SKIPPED: SnapshotManageAdminTest skipped as Neutron is not available
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_error_deleting [3.645502s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_update_snapshot_status [0.233317s] ... ok
{2} tempest.api.volume.admin.test_volume_hosts.VolumeHostsAdminTestsJSON.test_list_hosts [0.199641s] ... ok
{2} tempest.api.volume.admin.test_volume_hosts.VolumeHostsAdminTestsJSON.test_show_host [0.084746s] ... ok
{1} setUpClass (tempest.api.volume.admin.test_volume_manage.VolumeManageAdminTest) ... SKIPPED: Manage volume tests are disabled
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_delete_group_snapshots_following_updated_volumes [17.867160s] ... ok
{2} tempest.api.volume.admin.test_volume_pools.VolumePoolsAdminTestsJSON.test_get_pools_without_details [0.695590s] ... ok
{1} tempest.api.volume.admin.test_volume_quota_classes.VolumeQuotaClassesTest.test_show_default_quota [0.698545s] ... ok
{1} tempest.api.volume.admin.test_volume_quota_classes.VolumeQuotaClassesTest.test_update_default_quota [2.080089s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_delete_quota [1.657645s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_list_default_quotas [0.177100s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_list_quotas [0.170880s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_group_snapshot_create_show_list_delete [11.322543s] ... ok
{1} tempest.api.volume.admin.test_volume_retype.VolumeRetypeWithoutMigrationTest.test_available_volume_retype [5.565753s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_quota_usage [4.241410s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_quota_usage_after_volume_transfer [6.077732s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_show_quota_usage [0.212559s] ... ok
{1} tempest.api.volume.admin.test_volume_type_access.VolumeTypesAccessTest.test_volume_type_access_list [0.462178s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_update_all_quota_resources_for_tenant [0.768878s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsV319Test.test_reset_group_snapshot_status [9.577699s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_spec_create_get_delete [0.160808s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_specs_list [0.080016s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_disable_log_reason_with_no_reason [0.019348s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_disable_service_with_invalid_binary [0.026007s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_enable_service_with_invalid_host [0.072713s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_freeze_host_with_invalid_host [0.025716s] ... ok
{2} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_thaw_host_with_invalid_host [0.026475s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_specs_update [0.221153s] ... ok
{1} setUpClass (tempest.api.volume.admin.test_volumes_actions.VolumesActionsTest) ... SKIPPED: VolumesActionsTest skipped as Neutron is not available
{0} tempest.api.volume.admin.test_group_types.GroupTypesTest.test_group_type_create_list_update_show_delete [0.406107s] ... ok
{0} tempest.api.volume.admin.test_group_types.GroupTypesTest.test_group_type_list_by_optional_params [0.265537s] ... ok
{1} tempest.api.volume.test_extensions.ExtensionsTestJSON.test_list_extensions [0.108785s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_multi_backend.VolumeMultiBackendTest) ... SKIPPED: Cinder multi-backend feature disabled
{2} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_create_get_delete [0.260127s] ... ok
{2} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_encryption_create_get_update_delete [0.431426s] ... ok
{2} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_list [0.048801s] ... ok
{2} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_update [0.480541s] ... ok
{1} setUpClass (tempest.api.volume.test_image_metadata.VolumesImageMetadata) [0.000000s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 185, in setUpClass
    raise value.with_traceback(trace)

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 178, in setUpClass
    cls.resource_setup()

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_image_metadata.py", line 40, in resource_setup
    cls.volume = cls.create_volume(imageRef=CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}

{0} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_associate_disassociate_qos [0.718857s] ... ok
{0} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_back_end_consumer [0.186375s] ... ok
{0} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_both_consumer [0.162254s] ... ok
{0} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_front_end_consumer [0.139622s] ... ok
{0} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_get_qos [0.022153s] ... ok
{0} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_list_qos [0.024262s] ... ok
{0} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_set_unset_qos_key [0.154905s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_invalid_body [0.018070s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_none_body [0.015897s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_nonexistent_type_id [0.033535s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_delete_nonexistent_volume_type_id [0.019041s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_get_nonexistent_extra_spec_name [0.027680s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_get_nonexistent_volume_type_id [0.022380s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_list_nonexistent_volume_type_id [0.023638s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_multiple_extra_spec [0.018214s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_no_body [0.016618s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_none_extra_spec_id [0.041040s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_nonexistent_extra_spec_id [0.043251s] ... ok
{2} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_nonexistent_type_id [0.040371s] ... ok
{1} tempest.api.volume.test_snapshot_metadata.SnapshotMetadataTestJSON.test_crud_snapshot_metadata [0.296207s] ... ok
{1} tempest.api.volume.test_snapshot_metadata.SnapshotMetadataTestJSON.test_update_show_snapshot_metadata_item [0.247782s] ... ok
{0} tempest.api.volume.admin.test_user_messages.UserMessagesTest.test_delete_message [2.618200s] ... ok
{0} tempest.api.volume.admin.test_user_messages.UserMessagesTest.test_list_show_messages [2.021525s] ... ok
{2} tempest.api.volume.admin.test_volumes_list.VolumesListAdminTestJSON.test_volume_list_param_tenant [2.731508s] ... ok
{0} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_quota_volume_gigabytes [0.548380s] ... ok
{0} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_quota_volumes [0.581252s] ... ok
{1} tempest.api.volume.test_volume_transfers.VolumesTransfersV355Test.test_create_get_list_accept_volume_transfer [5.745417s] ... ok
{0} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_volume_extend_gigabytes_quota_deviation [0.505822s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_volume_retype.VolumeRetypeWithMigrationTest) ... SKIPPED: Cinder multi-backend feature disabled
{1} tempest.api.volume.test_volume_transfers.VolumesTransfersV355Test.test_create_list_delete_volume_transfer [4.058863s] ... ok
{2} tempest.api.volume.test_versions.VersionsTest.test_list_versions [0.013698s] ... ok
{2} tempest.api.volume.test_versions.VersionsTest.test_show_version [0.236584s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_host_name [0.026358s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_service_and_host_name [0.029175s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_service_binary_name [0.026867s] ... ok
{1} tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_bootable_volume [0.203380s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_clone.py", line 67, in test_create_from_bootable_volume
    src_vol = self.create_volume(imageRef=img_uuid)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 10:48:45,832 139434 INFO     [tempest.lib.common.rest_client] Request (VolumesCloneTest:test_create_from_bootable_volume): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.201s
2025-03-10 10:48:45,833 139434 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-031127ac-0f8f-426e-91f0-e4bed036ffa9', 'date': 'Mon, 10 Mar 2025 10:48:45 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_volume_host_name [4.067147s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_list_services [0.027778s] ... ok
{1} tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_volume [8.798656s] ... ok
{2} tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_delete_cascade [9.838236s] ... ok
{0} tempest.api.volume.admin.test_volume_snapshot_quotas_negative.VolumeSnapshotQuotasNegativeTestJSON.test_quota_volume_gigabytes_snapshots [1.149205s] ... ok
{0} tempest.api.volume.admin.test_volume_snapshot_quotas_negative.VolumeSnapshotQuotasNegativeTestJSON.test_quota_volume_snapshots [0.150960s] ... ok
{1} tempest.api.volume.test_volumes_clone_negative.VolumesCloneNegativeTest.test_create_from_volume_decreasing_size [4.015228s] ... ok
{1} setUpClass (tempest.api.volume.test_volumes_extend.VolumesExtendAttachedTest) ... SKIPPED: VolumesExtendAttachedTest skipped as Neutron is not available
{2} tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_from_snapshot_cascade_delete [10.459650s] ... ok
{0} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_volume_type_encryption_nonexistent_type_id [0.200372s] ... ok
{0} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_with_empty_name [0.016539s] ... ok
{0} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_with_repeated_name [0.148712s] ... ok
{0} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_delete_nonexistent_type_id [0.019104s] ... ok
{0} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_get_nonexistent_type_id [0.018872s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete [7.760025s] ... ok
{0} tempest.api.volume.test_availability_zone.AvailabilityZoneTestJSON.test_get_availability_zone_list [0.077681s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersTest.test_create_get_list_accept_volume_transfer [6.289210s] ... ok
{0} tempest.api.volume.test_volume_absolute_limits.AbsoluteLimitsTests.test_get_volume_absolute_limits [0.212747s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersTest.test_create_list_delete_volume_transfer [4.022143s] ... ok
{2} setUpClass (tempest.api.volume.test_volumes_actions.VolumesActionsTest) ... SKIPPED: VolumesActionsTest skipped as Neutron is not available
{0} tempest.api.volume.test_volume_metadata.VolumesMetadataTest.test_crud_volume_metadata [0.356601s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_as_clone [16.945270s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_from_image [0.020843s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_get.py", line 132, in test_volume_create_get_update_delete_from_image
    image = self.images_client.show_image(CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 10:49:29,455 139434 INFO     [tempest.lib.common.rest_client] Request (VolumesGetTest:test_volume_create_get_update_delete_from_image): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.018s
2025-03-10 10:49:29,455 139434 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-9e29e968-65c1-4dad-bd95-e1cf6e3d4a51', 'date': 'Mon, 10 Mar 2025 10:49:29 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{0} tempest.api.volume.test_volume_metadata.VolumesMetadataTest.test_update_show_volume_metadata_item [0.259138s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_from_deactivated_image [0.562531s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_from_image_with_decreasing_size [0.447159s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_invalid_size [0.017558s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_snapshot_id [0.028803s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_source_volid [0.030806s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_volume_type [0.034938s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_negative [0.017413s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_zero [0.018551s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_without_passing_size [0.016214s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_delete_invalid_volume_id [0.026443s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_delete_volume_without_passing_volume_id [0.014038s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_detach_volumes_with_invalid_volume_id [0.033548s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_get_invalid_volume_id [0.027141s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_get_volume_without_passing_volume_id [0.010983s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_detail_with_invalid_status [0.048045s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_detail_with_nonexistent_name [0.049764s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_with_invalid_status [0.033648s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_with_nonexistent_name [0.035243s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_reserve_volume_with_negative_volume_status [0.132061s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_reserve_volume_with_nonexistent_volume_id [0.025653s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_unreserve_volume_with_nonexistent_volume_id [0.026298s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_empty_volume_id [0.011554s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_invalid_volume_id [0.028395s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_nonexistent_volume_id [0.028156s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_delete_nonexistent_volume_id [0.026283s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_None_size [0.015929s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_non_number_size [0.016059s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_nonexistent_volume_id [0.031465s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_size_smaller_than_original_size [0.015468s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_without_passing_volume_id [0.029362s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_get_nonexistent_volume_id [0.026277s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesSummaryTest.test_show_volume_summary [0.197678s] ... ok
{2} setUpClass (tempest.api.volume.test_volumes_snapshots.VolumesSnapshotTestJSON) ... SKIPPED: VolumesSnapshotTestJSON skipped as Neutron is not available
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersV357Test.test_create_get_list_accept_volume_transfer [5.617088s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list [0.041713s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_by_name [0.036028s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_by_name [0.052824s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_pagination [0.247501s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_with_multiple_params [0.114695s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_pagination [0.166691s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_param_display_name_and_status [0.035528s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_display_name_and_status [0.056358s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_marker [0.060209s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_metadata [0.066206s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_details [0.088317s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_param_metadata [0.043350s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_availability_zone [0.329907s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_bootable [0.070012s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_status [0.069592s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_availability_zone [0.056520s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_bootable [0.060620s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_status [0.062538s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit [0.031338s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit_equals_infinite [0.058092s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit_equals_zero [0.025813s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_marker [0.067741s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_offset [0.028108s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_created_at_asc [0.028896s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_created_at_desc [0.026497s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_id_asc [0.029941s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_id_desc [0.032011s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_name_asc [0.029859s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_name_desc [0.028742s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshots_list_details_with_params [0.094048s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshots_list_with_params [0.088838s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersV357Test.test_create_list_delete_volume_transfer [3.970462s] ... ok
{0} tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend [0.199844s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_extend.py", line 36, in test_volume_extend
    volume = self.create_volume(imageRef=self.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 10:49:50,871 139432 INFO     [tempest.lib.common.rest_client] Request (VolumesExtendTest:test_volume_extend): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.196s
2025-03-10 10:49:50,871 139432 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-726b56d2-b04f-4ac4-afd5-d69a89460283', 'date': 'Mon, 10 Mar 2025 10:49:50 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_create_snapshot_with_nonexistent_volume_id [0.213994s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_create_snapshot_without_passing_volume_id [0.032199s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshot_invalid_param_limit [0.014030s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshots_invalid_param_marker [0.027227s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshots_invalid_param_sort [0.017183s] ... ok
{0} tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend_when_volume_has_snapshot [8.694013s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_volume_from_snapshot_decreasing_size [7.720219s] ... ok

==============================
Failed 4 tests - output below:
==============================

setUpClass (tempest.api.volume.test_image_metadata.VolumesImageMetadata)
------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 185, in setUpClass
    raise value.with_traceback(trace)

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 178, in setUpClass
    cls.resource_setup()

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_image_metadata.py", line 40, in resource_setup
    cls.volume = cls.create_volume(imageRef=CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_bootable_volume[id-cbbcd7c6-5a6c-481a-97ac-ca55ab715d16,image]
--------------------------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_clone.py", line 67, in test_create_from_bootable_volume
    src_vol = self.create_volume(imageRef=img_uuid)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 10:48:45,832 139434 INFO     [tempest.lib.common.rest_client] Request (VolumesCloneTest:test_create_from_bootable_volume): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.201s
2025-03-10 10:48:45,833 139434 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-031127ac-0f8f-426e-91f0-e4bed036ffa9', 'date': 'Mon, 10 Mar 2025 10:48:45 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'


tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_from_image[id-54a01030-c7fc-447c-86ee-c1182beae638,image,smoke]
-------------------------------------------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_get.py", line 132, in test_volume_create_get_update_delete_from_image
    image = self.images_client.show_image(CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 10:49:29,455 139434 INFO     [tempest.lib.common.rest_client] Request (VolumesGetTest:test_volume_create_get_update_delete_from_image): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.018s
2025-03-10 10:49:29,455 139434 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-9e29e968-65c1-4dad-bd95-e1cf6e3d4a51', 'date': 'Mon, 10 Mar 2025 10:49:29 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'


tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend[id-9a36df71-a257-43a5-9555-dc7c88e66e0e]
--------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_extend.py", line 36, in test_volume_extend
    volume = self.create_volume(imageRef=self.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-03-10 10:49:50,871 139432 INFO     [tempest.lib.common.rest_client] Request (VolumesExtendTest:test_volume_extend): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.196s
2025-03-10 10:49:50,871 139432 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-726b56d2-b04f-4ac4-afd5-d69a89460283', 'date': 'Mon, 10 Mar 2025 10:49:50 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'



======
Totals
======
Ran: 203 tests in 203.0036 sec.
 - Passed: 189
 - Skipped: 10
 - Expected Fail: 0
 - Unexpected Success: 0
 - Failed: 4
Sum of execute time for each test: 271.5881 sec.

==============
Worker Balance
==============
 - Worker 0 (49 tests) => 0:03:20.289950
 - Worker 1 (58 tests) => 0:03:03.510582
 - Worker 2 (96 tests) => 0:03:20.351815
